test/plots/piePlotTests.ts

```diff
@@ -165,6 +165,31 @@ describe("Plots", () => {
         svg.remove();
       });
 
+      it("entityAt with defaults", () => {
+        var click1 = { x: 300, y: 200 };
+        var entity1 = piePlot.entitiesAt(click1);
+        TestMethods.assertPlotEntitiesEqual(entity1[0], piePlot.entities()[0], "entities are equal");
+        var click2 = { x: 200, y: 300 };
+        var entity2 = piePlot.entitiesAt(click2);
+        TestMethods.assertPlotEntitiesEqual(entity2[0], piePlot.entities()[1], "entities are equal");
+        var click3 = { x: 0, y: 0 };
+        var entity3 = piePlot.entitiesAt(click3);
+        assert.strictEqual(entity3.length, 0, "no entities returned");
+        svg.remove();
+      });
+
+      it("entityAt with radius modifications", () => {
```

Comment:
Same naming stuff here. Maybe something like "points within innerRadius() or outside of outerRadius() don't return any entities"?